spanner_orm/field.py

```diff
@@ -43,23 +43,43 @@ def grpc_type() -> spanner_v1.Type:
   def validate_type(value: Any) -> None:
     raise NotImplementedError
 
+  @staticmethod
+  def matches(type: str) -> bool:
+    raise NotImplementedError
+
+  @staticmethod
+  def support_length() -> bool:
+    raise NotImplementedError
```

Comment:
(nit) I think these should either have `@abc.abstractmethod` so they have to be overridden, or they should return sensible defaults instead of raising NotImplementedError. I'm currently leaning towards support_length() returning False by default. Not sure about matches(), I could either go with a default of `return type == self.ddl()` or the abstract decorator.

Reply:
Added @abc.abstractmethod to matches so that this must be overriden. For supports_length() I let it return False by default.